tests/migrations/test_loader.py

```diff
@@ -202,6 +202,23 @@ def test_marked_as_unmigrated(self):
         self.assertEqual(migration_loader.migrated_apps, set())
         self.assertEqual(migration_loader.unmigrated_apps, {'migrated_app'})
 
+    @override_settings(
+        INSTALLED_APPS=['migrations.migrations_test_apps.migrated_app'],
+    )
+    def test_disable_migrations(self):
+        connections = ConnectionHandler({
+            'default': {
+                'NAME': 'dummy',
```

Comment:
This leaves a "dummy" SQLite database on the filesystem after running the tests.